jgt_tools/check_version.py

```diff
@@ -11,14 +11,32 @@ def _version_changed(pyproject_diff):
     return "+version = " in pyproject_diff
 
 
+def find_default_branch():
+    """Find the default branch for a git repository."""
+    remotes = subprocess.check_output(
+        ["git", "remote"], universal_newlines=True
+    ).splitlines()
+    if len(remotes) == 1:
+        remote = remotes[0]
+    elif "origin" in remotes:
+        remote = "origin"
+    else:
+        raise KeyError("Cannot determine remote")
```

Comment:
NB: If you had a separate "find remote" helper the code could be slightly simpler:
```
def find_remote():
    remotes = ...
    if len(remotes) == 1:
        return remotes[0]
    if "origin" in remotes:
        return "origin"
    raise KeyError(...)
```